common/src/main/java/org/opensearch/ml/common/CommonValue.java

```diff
@@ -262,13 +262,13 @@ public class CommonValue {
                         + MLModel.LAST_UNDEPLOYED_TIME_FIELD
                         + "\": {\"type\": \"date\", \"format\": \"strict_date_time||epoch_millis\"},\n"
                         + "      \""
+                        + MLModel.GUARDRAILS_FIELD
```

Comment:
Have you verified if the index mapping is valid ? You can manually try it in OSD dev tools